lib/node_modules/@stdlib/math/stats/anova1/lib/index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**
+* One-Way analysis of variance.
+*
+* @module @stdlib/math/stats/anova1
+*
+* @example
+* var anova1 = require( '@stdlib/math/stats/anova1' );
+* var x = [1, 3, 5, 2, 4, 6, 8, 7, 10, 11, 12, 15];
+* var f = ['Frank', 'Nugent', 'Philipp', 'Gordon', 'Frank', 'Nugent', 'Philipp', 'Gordon', 'Frank', 'Nugent', 'Philipp', 'Gordon' ];
+*
+* var out = anova1(x, f);
+* // returns { treatment_df: 11,
```

Comment:
This should be a multi-line comment (starting with `/*`). We parse the example code in order to test that it executes, and right now this won't work here as L15-L36 are not part of a comment but mistakenly seen as code. Let's also use quotation marks around object keys.

Reply:
Resolved.